src/Manifolds.jl

```diff
@@ -99,60 +64,68 @@ using HybridArrays
 using LinearAlgebra
 using LightGraphs
```

Comment:
I thought that just `using Package` would bring all its public functions to local scope, so why do we do a using and then import specific ones separately again?

Reply:
I think you mean `using`? Yeah, that's a carry-over from the older code. Having now looked at LightGraphs, I see that `AbstractGraph` is exported, so we don't need to "use" it.

Reply:
We could check for ManifoldsBase, too – since we use the full package, I think most specific function/type uses can be omitted?